src/component/variantPage/functionalPrediction/AlphaMissense.tsx

```diff
@@ -0,0 +1,232 @@
+                >
+                    here
+                </a>
+                .
+            </div>
+        </div>
+    ) : null;
+};
+
+@observer
+export default class AlphaMissense extends React.Component<
+    IAlphamissenseProps,
+    {}
+> {
+    @observable showDetails = false;
+
+    constructor(props: IAlphamissenseProps) {
+        super(props);
+        makeObservable(this);
+    }
+
+    public render() {
+        let alphamissenseContent: JSX.Element;
+
+        const dataSource = (
+            <>
+                AlphaMissense&nbsp;
+                <i className="fas fa-external-link-alt" />
+            </>
+        );
+        if (this.props.amClass === 'N/A') {
+            alphamissenseContent = <span> N/A </span>;
+        } else if (this.props.amClass && this.props.amClass.length > 0) {
+            alphamissenseContent = (
+                <span>
+                    <p>
+                        {this.props.amClass + ' '}(
+                        {this.props.amPathogenicityScore})
+                    </p>{' '}
+                </span>
+            );
+        } else {
+            alphamissenseContent = <span> N/A </span>;
+        }
+
+        return (
+            <div className={functionalGroupsStyle['functional-group']}>
+                <div className={functionalGroupsStyle['data-source']}>
+                    <DefaultTooltip
+                        placement="top"
+                        overlay={
+                            <div style={{ maxWidth: 450 }}>
+                                <AlphamissenseInfo />
+                            </div>
+                        }
+                    >
+                        <a
+                            href={ALPHAMISSENSE_URL}
+                            target="_blank"
+                            rel="noopener noreferrer"
+                        >
+                            {dataSource}
+                        </a>
+                    </DefaultTooltip>
+                </div>
+                <div>
+                    <span className={functionalGroupsStyle['data-with-link']}>
+                        <a
+                            href={ALPHAMISSENSE_URL}
+                            target="_blank"
+                            rel="noopener noreferrer"
+                        >
+                            {alphamissenseContent}
+                        </a>
+                    </span>
+                    <Collapse in={this.showDetails}>
+                        <div className="pt-2">
+                            <AlphamissenseValue
+                                amClass={this.props.amClass}
+                                amPathogenicityScore={
+                                    this.props.amPathogenicityScore
+                                }
+                            />
+                            <br />
+                            <AlphamissenseLegend />
+                        </div>
+                    </Collapse>
```

Comment:
I think this content is always hidden because we don't seem to have a `More` button

Reply:
deleted <Collapse>